tests/processes/ProcessingTest.py

```diff
@@ -131,6 +131,13 @@ def test_run(self):
                          ') at 0x[0-9a-fA-F]+>'.format(hex(global_result.id)))
 
     def test_empty_run(self):
+        execute_section(self.sections['default'],
+                        [],
+                        [],
+                        lambda *args: self.result_queue.put(args[2]),
+                        None,
+                        self.log_printer,
+                        console_printer=self.console_printer)
```

Comment:
@Mixih Yes, but I'm afraid I can't do that. The part that is not covered is in a try-catch statement.
The following is from Processing.py. The line after `except IndexError` was not covered.
```
try:
        running_processes = int(section['jobs'])
    except ValueError:
        log_printer.warn("Unable to convert setting 'jobs' into a number. "
                         'Falling back to CPU count.')
        running_processes = get_cpu_count()
    except IndexError:
        running_processes = get_cpu_count()
```
So even if i trigger the index error, the caller cannot catch the exception as it is already taken care of in the above catch. Else I'll have to remove the try-catch statements.